src/tower_arduino_ide/relay_bus.cpp

```diff
@@ -0,0 +1,24 @@
+#include "relay_bus.h"
+
+void relay_bus_init() {
+	Wire.begin();
+}
+
+uint8_t relay_bus_write(uint8_t addr, uint8_t data) {
+	Wire.beginTransmission(addr);	// transmit to device #4
+	Wire.write(data);        		// sends the 1 byte of data
+	Wire.endTransmission();			// stop transmitting
+	return 1;

```

Comment:
what are you hoping to get out of this return code? If you don't actually return different codes, why not make that function void? (Also, nitpick: in C the convention is to return 0 on success. Non-zero return codes in functions that return an `int` are typically errors.)

Reply:
Yea this was just a placeholder when I was coding, will probably remove it later